app/controllers/sessions_controller.rb

```diff
@@ -52,4 +53,15 @@ def set_csrf_headers
     response.headers['X-CSRF-Param'] = request_forgery_protection_token.to_s
     response.headers['X-CSRF-Token'] = form_authenticity_token.to_s
   end
+
+  def revoke_access_tokens!
+    application_id_to_revoke = doorkeeper_token&.application_id
```

Comment:
the doorkeeper token should exist here due to the check on token existence before you call this method, either remove the safe operator or add a guard clause first thing to this method
```
return unless doorkeeper_token
```